compiler/main/commonFlags.h

```diff
@@ -65,6 +65,9 @@ void driverSetDevelSettings(const ArgumentDescription* desc, const char* arg_unu
 #define DRIVER_ARG_LICENSE \
   {"license", ' ', NULL, "Show license", "F", &fPrintLicense, NULL, NULL}
 
+#define DRIVER_ARG_HOME \
+  {"home", ' ', "<path>", "Path to Chapel's home directory", "S", NULL, "_CHPL_HOME", setHome}
```

Comment:
No, `arg.h` doesn't include anything, so there's no circularity.  I suspect the reason that this doesn't require `#include "driver.h"` is that all files that `#include` this file and use the macro also `#include "driver.h"` so by the time all the #includes are inlined and the macros expanded, all necessary symbols are visible.  We should be able to add `#include "driver.h"` here if desired.  I tend to take a "only add the `#include`s that are needed to keep things compiling approach when I develop (for better or worse), which is why I didn't.

Reply:
I'm going to merge to try and clear up the standing failure, and we can consider this in a separate PR if you want to advocate for it.

Reply:
That seems reasonable.  I do think it's probably safer to add the `#include`, in case other compilers are more strict about it or become more strict in the future.  It seems justifiable since it's obviously a symbol that we get from that header file

Reply:
I will look into adding the `#include`, but to stave off some paranoia: I'd warrant that there's no chance of this changing in the future (i.e., for `#include` to mean something other than "have the C pre-processor inline the contents of this file in here").  This is how cpp has worked since the dark ages, and if it were to change, the world of C programs would crumble and fall.

Reply:
Actually, maybe a better approach would be to just move the `setHome` prototype to `commonFlags.h` since that's where it's needed and it's `#include`d in both places?

Reply:
I'd be okay with that approach as well!